sound/hda/intel-dsp-config.c

```diff
@@ -472,6 +472,14 @@ static const struct config_entry config_table[] = {
 	},
 #endif
 
+/* Meteor Lake */
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_METEORLAKE)
+	{
```

Comment:
A small suggestion, it is recommended to add /* Meteorlake-P */ before the opening brace.